package.json

```diff
@@ -63,8 +63,12 @@
     "assert-rejects": "^1.0.0",
     "c8": "^7.1.0",
     "codecov": "^3.6.5",
+    "eslint": "^6.8.0",
```

Comment:
You can drop all of the `eslint` dependencies

Reply:
removed `eslint` related dependencies.